torchgen/model.py

```diff
@@ -728,6 +756,7 @@ def has_composite_kernel(self) -> bool:
         return (
             self.has_composite_implicit_autograd_kernel
             or self.has_composite_explicit_autograd_kernel
+            or len(self.composite) > 0
```

Comment:
No action needed, but this semantics of this function are kind of confusing. Today at least, it really means "has an explicitly mentioned `CompositeImplicitAutograd/CompositeExplicitAutograd` entry in `native_functions.yaml`.

It seems reasonable to have it mean "does this op get a composite kernel registered to the dispatcher", although it doesn't really do that today, since the codegen will generate `CompositeExplicitAutograd` kernels for all `functional`/`inplace` structured operators.